services/postprocessing/README.md

```diff
@@ -99,3 +99,5 @@ ocis postprocessing restart                # Restarts all uploads where postproc
 ocis postprocessing restart -s "finished"  # Equivalent to the above
 ocis postprocessing restart -s "virusscan" # Restart all uploads currently in virusscan step
 ```
+
+Note: All above commands containting the word `restart` can also be replaced by `resume`. This changes behaviour slightly. When `restarting` an upload, the whole postprocessing will be (re)done. If `resuming` an upload it will only continue from the last step that was completed.
```

Comment:
```suggestion
Note: All above commands containting the word `restart` can also be replaced by `resume`. This changes behaviour slightly. When `restarting` an upload, the whole postprocessing will be (re)done. If `resuming` an upload, it will only continue from the last step that was completed.
```